Core/Src/main.c

```diff
@@ -32,6 +32,10 @@
 /* Private define ------------------------------------------------------------*/
 /* USER CODE BEGIN PD */
 #define TX_ID 0x440
+#define HSP_SERVO_MIN_PULSE_WIDTH 500
+#define HSP_SERVO_MAX_PULSE_WIDTH 2500
+#define HSP_SERVO_PWM_PERIOD 20000
+#define HSP_SERVO_MAX_DEG 270

```

Comment:
works for now, but eventually will be changed with flash memory right? other option is just store an array here instead of the value and have it check against servo board

Reply:
This one could be put into flash memory eventually too, but there'd be no reason in checking it against the servo board since these vals have to do with the servo not the board.  Board doesn't know what kinda servo is connected to it.